Source/WebCore/contentextensions/NFAToDFA.cpp

```diff
@@ -287,7 +287,7 @@ struct DataConverterWithEpsilonClosure {
             auto addResult = destination.add(nodeId);
             if (addResult.isNewEntry) {
                 const UniqueNodeList& nodeClosure = nfaNodeclosures[nodeId];
-                destination.add(nodeClosure.begin(), nodeClosure.end());
+                destination.addAll(nodeClosure);
```

Comment:
No need for the local variable.
```suggestion
                destination.addAll(nfaNodeclosures[nodeId]);
```